citesphere/src/main/java/edu/asu/diging/citesphere/core/repository/custom/AuthorityRepository.java

```diff
@@ -21,5 +22,9 @@
 
     public long countByUsernameAndNameContainingAndNameContainingOrderByName(@Param("username") String username,
             @Param("firstName") String firstName, @Param("lastName") String lastName);
+    
+    @Query("select a from AuthorityEntry a where :group member of a.groups and a.name like %:firstName% and a.name like %:lastName% order by a.name")
+    public List<IAuthorityEntry> findByGroupAndFirstNameAndLastName(@Param("group") Long group,
```

Comment:
have you tried using the automatic query generation here instead of writing the query?

Reply:
When I tried to search for a translation for "member of", I didn't find any. Turns out "Containing" also works for this.